src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php

```diff
@@ -495,6 +495,8 @@ public static function provideTitle()
             ['Symfony', 'symfony', false],
             ['Symfony', 'Symfony', false],
             ['SYmFOny', 'sYmFOny', false],
+            ['¡Hola spain!', '¡hola spain!', false],
+            ['¡Hola Spain!', '¡hola spain!', true],
```

Comment:
Can you also add a test case where the first letter is one with an accent?

Reply:
Done!

Reply:
I can't see this, where?
Here is an example: `être ou ne pas être`

Reply:
Sorry I didn't understand you for the first time. Added.